deps/v8/tools/gen-postmortem-metadata.py

```diff
@@ -139,14 +167,31 @@
         'value': 'ScopeInfo::kStackLocalCount' },
     { 'name': 'scopeinfo_idx_ncontextlocals',
         'value': 'ScopeInfo::kContextLocalCount' },
+    { 'name': 'scopeinfo_idx_ncontextglobals',
+        'value': 'ScopeInfo::kContextLocalCount' },
```

Comment:
@bnoordhuis do you think it may be a good idea to fix it here before the proper fix will land in V8?